app/menu.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import { openExternal } from './utils/open-external';
+import packageJson from '../package.json';
+
+const DOCS_URL = 'https://zepiowallet.com/';
```

Comment:
This is great. I'm going to see what makes sense to be part of the core Menu so hold tight for now and don't merge yet! 👍

Reply:
Great!